app/components/Map.js

```diff
@@ -181,7 +181,7 @@ export default class Map extends Component {
         <MapView
           ref="map"
           showsUserLocation={true}
-          initialRegion={{ longitudeDelta: 0.005, latitude: currLoc.latitude,longitude: currLoc.longitude, latitudeDelta: 0.005 }}
+-         initialRegion={{ longitudeDelta: 0.005, latitude: currLoc.latitude,longitude: currLoc.longitude, latitudeDelta: 0.005 }}
```

Comment:
I found the same thing in the previous PR. I was just about to do a bugfix for this. 
So, we should take it out for now and I'll submit PR to fix.